src/App.js

```diff
@@ -9,40 +9,41 @@ import Input from './components/Input';
 import Button from './components/Button';
 import List from './components/List';
 import Counter from './components/Counter';
+import ErrorMessage from './components/ErrorMessage';
 import { addTask, removeTask } from './store/actions/listActions';
 
 class App extends Component {
   state = {
     value: '',
-    isInputEmpty: false,
-    isValueExist: false,
+    isError: false,
+    errorMessage: '',
   };
 
-  addTask = () =>{
+  addTask = () => {
     const { value } = this.state;
     const { tasks, addTask } = this.props;
     if (isEmpty(value)) {
-      this.setState({ isInputEmpty: true })
+      this.setState({ isError: true, errorMessage: "Task can't be empty" })
```

Comment:
good approach is to have separate file with constants containing error messages.